www/src/components/search-form.js

```diff
@@ -306,6 +306,14 @@ class SearchForm extends Component {
       document.head.appendChild(link)
     }
   }
+  componentWillUnmount() {
+    loadedJs = false
```

Comment:
This doesn't need undone as `loadedJs` is local to the component so will get removed when the component is removed.

Reply:
`loadedJs ` is in module scope and not component state/field so it will be preserved between remounts I think

Reply:
It doesn't work without setting loadedJs back to false. I strongly guess that's because it's declared outside of the component? https://github.com/gatsbyjs/gatsby/blob/2067b6ec248ba446d63cd5d0b007b25b25d3596d/www/src/components/search-form.js#L11

Reply:
Oh @pieh thank you for commenting, didn't refresh page. Yeah that sounds a little more informed than my guessplaining :P

Reply:
oops, sorry for the sloppy review 😬

Reply:
🤗